.env.example

```diff
@@ -4,4 +4,7 @@ DATABASE_PASSWORD=postgres
 DATABASE_PORT=5432
 DATABASE_NAME=postgres
 
-STATIC_WORKER_POOL=5
\ No newline at end of file
+STATIC_WORKER_POOL=5
```

Comment:
what is this STATIC_WORKER_POOL? how this is related to the arrebol new design?

Reply:
It is legacy. I removed.